app/util/app-insights-util.js

```diff
@@ -0,0 +1,22 @@
+const appInsights = require('applicationinsights')
+
+module.exports = {
+  setSessionId: (correlationId) => {
+    if (appInsights.defaultClient !== null && appInsights.defaultClient !== undefined) {
```

Comment:
Do we need to check for `null` too or would `undefined` be sufficient?  It what scenario would it be `null` as opposed to `undefined`?

Reply:
Removed null check as undefined is sufficient.